datastore/longhorn.go

```diff
@@ -930,30 +930,51 @@ func (s *DataStore) CreateNode(node *longhorn.Node) (*longhorn.Node, error) {
 	return ret, nil
 }
 
-// CreateDefaultDisk creates a default DiskSpec based on DefaultDataPath and applies it to the provided Node.
-func (s *DataStore) CreateDefaultDisk(node *longhorn.Node) error {
-	pathSetting, err := s.GetSetting(types.SettingNameDefaultDataPath)
-	if err != nil {
-		return err
-	}
+func (s *DataStore) CreateDefaultDisks(node *longhorn.Node, annotations map[string]string) error {
```

Comment:
We should move `CreateDefaultDisks` etc from datastore if we can. Those contains too many logics, and out of the scope of a simple datastore.

Reply:
I will move `getDisksFromAnnotation()` and `getNodeTagsFromAnnotation()` to package `type`. As for `CreateDefaultDisks`, I am not sure where I can move it to

Reply:
Node controller?

Reply:
it is also used in `datastore.CreateDefaultNode()` hence NodeController is not a good choice.

Reply:
Can we move CreateDefaultNode as well?

Reply:
Let me check and try it

Reply:
We can move `CreateDefaultNode()` only. 
The function `CreateDefaultNode()` involves in `*longhorn.Node`. And it cannot be used in the package `types` due to the recursive import issue.